perfect_share.py

```diff
@@ -0,0 +1,35 @@
+def is_prime(num):
+    if num > 1:
+        for n in range(2, num):
+            if (num % n) == 0:
+                return False
+        return True
+    else:
+        return False
+
+
+def is_mersenne_prime(prime, mersenne_prime):
```

Comment:
Note that most of the code readers wont know what is a mesenne_prime.
Also, I guess that you are about to use an euclidean algorithm in the next function to find the perfect numbers.
I think that it is great, but without documentation (and an explanation or a link to one) it will be very hard to understand.